qa/workunits/windows/test_rbd_wnbd.py

```diff
@@ -376,36 +382,32 @@ def cleanup(self):
     @Tracer.trace
     @retry_decorator()
     def _init_disk(self):
-        cmd = ("powershell.exe", "-command",
-               f"Get-Disk -Number {self.disk_number} | "
+        cmd = (f"Get-Disk -Number {self.disk_number} | "
                "Initialize-Disk")
-        execute(*cmd)
+        ps_execute(cmd)
 
     @Tracer.trace
     @retry_decorator()
     def _create_partition(self):
-        cmd = ("powershell.exe", "-command",
-               f"Get-Disk -Number {self.disk_number} | "
+        cmd = (f"Get-Disk -Number {self.disk_number} | "
                "New-Partition -AssignDriveLetter -UseMaximumSize")
-        execute(*cmd)
+        ps_execute(cmd)
 
     @Tracer.trace
     @retry_decorator()
     def _format_volume(self):
         cmd = (
-            "powershell.exe", "-command",
-             f"(Get-Partition -DiskNumber {self.disk_number}"
-             " | ? { $_.DriveLetter }) | Format-Volume -Force -Confirm:$false")
-        execute(*cmd)
+            f"(Get-Partition -DiskNumber {self.disk_number}"
```

Comment:
Nit: This could go after `cmd = (`, on the same line.

Reply:
The second line would exceed 80 characters:

```
def _format_volume(self):
        cmd = (f"(Get-Partition -DiskNumber {self.disk_number}"
               " | ? { $_.DriveLetter }) | Format-Volume -Force -Confirm:$false")
```

The pep8 linter wouldn't be happy with the following indentation:

```
def _format_volume(self):
        cmd = (f"(Get-Partition -DiskNumber {self.disk_number}"
            " | ? { $_.DriveLetter }) | Format-Volume -Force -Confirm:$false")
```

We'd have to either keep this or add a third line:

```
def _format_volume(self):
        cmd = (f"(Get-Partition -DiskNumber {self.disk_number}"
               " | ? { $_.DriveLetter }) | "
               "Format-Volume -Force -Confirm:$false")
```

That's why I usually prefer starting long multi-line strings from a new line, so that I get a little more headroom for the string by avoiding some extra indents.

Reply:
> The second line would exceed 80 characters:

My apologies, I commented on the wrong line.  This comment was supposed to go for the one in `_get_drive_letter`.  This one, in `_format_volume`, was better with your original formatting.

Do you want to revert this hunk and amend the one in `_get_drive_letter` or just punt on this?

Reply:
`make check` needs to be rerun anyway -- might as well be after another force push ;)